apis/v1alpha1/suite_test.go

```diff
@@ -54,24 +53,14 @@ func TestAPIs(t *testing.T) {
 }
 
 var _ = BeforeSuite(func(done Done) {
-
 	logf.SetLogger(zap.New(zap.UseDevMode(true), zap.WriteTo(GinkgoWriter)))
 
-	// We are running in bazel so set up the directory for the test binaries
-	if os.Getenv("TEST_WORKSPACE") != "" {
-		// TODO change these by splitting the args that we are passing in
-		paths.MaybeSetEnv("TEST_ASSET_ETCD", "etcd", "hack", "bin", "etcd")
-		paths.MaybeSetEnv("TEST_ASSET_KUBE_APISERVER", "kube-apiserver", "hack", "bin", "kube-apiserver")
-		paths.MaybeSetEnv("TEST_ASSET_KUBECTL", "kubectl", "hack", "bin", "kubectl")
-	}
-
 	By("bootstrapping test environment")
-	testEnv = &envtest.Environment{
-		CRDDirectoryPaths: []string{filepath.Join("..", "..", "config", "crd", "bases")},
-	}
+	testEnv = env.NewEnv(runtime.NewSchemeBuilder(AddToScheme), filepath.Join("..", "..", "config", "crd", "bases"))
 
 	var err error
-	cfg, err = testEnv.Start()
+	testEnv.Start()
```

Comment:
why stop checking err here? delete line 64 if we want to keep it this way

Reply:
Our implementation of `Env.Stop()` actually doesn't return an error. It prints out an error if one occurs, and panics if it fails to uninstall CRDs.